integration-test/src/main/java/com/sequenceiq/it/cloudbreak/cloud/v4/azure/AzureCloudProvider.java

```diff
@@ -280,25 +280,8 @@ public StackAuthenticationTestDto stackAuthentication(StackAuthenticationTestDto
         return stackAuthenticationEntity.withPublicKey(sshPublicKey);
     }
 
-    public String getNetworkId() {
-        return azureProperties.getNetwork().getNetworkId();
-    }
-
-    public Set<String> getSubnetIds() {
-        return azureProperties.getNetwork().getSubnetIds();
-    }
-
-    public String getSubnetId() {
-        Set<String> subnetIDs = getSubnetIds();
-        return subnetIDs.iterator().next();
-    }
-
-    public Boolean getNoPublicIp() {
-        return azureProperties.getNetwork().getNoPublicIp();
-    }
-
-    public String getResourceGroupName() {
-        return azureProperties.getNetwork().getResourceGroupName();
+    public Set<String> getSubnetIds(boolean usePrivateEndpointCapableNetwork) {
```

Comment:
We can remove this as well with the above methods, because of `usePrivateEndpointCapableNetwork` is not needed anymore.
So we can use directly `azureProperties.getNetwork().getSubnetIds().stream().findFirst().get()` or `azureProperties.getNetwork().getSubnetIds()`